### ISSUE TITLE
Capslock key only useful when VoiceOver is Disabled

### ISSUE THREAD

**Issue by a-singer** [EscolarProgramming/NVDARemotePublic#3]
Hello,

When connected to the remote server and VoiceOver is enabled, I cannot use the capslock key as the NVDA modifier since it is used by VO as its VO key. I have to turn off voiceOver to get the capslock key to work as an NVDA key. By turn off, I mean triple click home to get VO completely off. Would it be possible to set another key as the NVDA key, perhaps as an option? Maybe the right alt key or something like it could be used so that for NVDA commands, I could send right alt+command where I could still use capslock for VO? I know that one can use alt+ctrl as the VO keys, but alt+ctrl is also used on the computer for shortcuts, and it is a point of friction to switch back and forth between capslock and alt+ctrl whenever one wishes to use NVDA remote. If NVDA remote could have an option to turn off capslock as the VO key when in the program, that, too, would work very well.

Thanks,

**Comment by EscolarProgramming** [comment]
Hello,
This is nothing I can change, you can set the NVDA key by your own. Rightclick NVDA -> Settings -> Keyboard settings (maybe the name is a bit different, I have to translate the menu), an there you can configure the key.

**Comment by ChrisDuffley** [comment]
I did not think you could change this on VO's end. My best advice is to use an activity in VoiceOver settings whenever NVDARemote is open which would then disable the caps lock key. You may just have to disable VoiceOver for now in order to get this to work correctly as well.
